overwatch-rs/src/services/relay.rs

```diff
@@ -141,7 +141,7 @@ impl<S: ServiceCore> Relay<S> {
     }
 
     #[instrument(skip(self), err(Debug))]
-    pub async fn connect(&self) -> Result<OutboundRelay<S::Message>, RelayError> {
+    pub async fn connect(self) -> Result<OutboundRelay<S::Message>, RelayError> {
```

Comment:
This should probably be other PR. But it should be ok this time as it is really simple. Just add the change to the PR title.